src/components/Accordion/Accordion.tsx

```diff
@@ -61,20 +62,26 @@ export const Accordion = (props: AccordionProps): React.ReactElement => {
     'usa-accordion',
     {
       'usa-accordion--bordered': bordered,
+      'aria-multiselectable': multiselectable,
     },
     className
   )
 
   const toggleItem = (itemId: AccordionItem['id']): void => {
-    const newOpenItems = [...openItems]
+    let newOpenItems = [...openItems]
     const itemIndex = openItems.indexOf(itemId)
+    const isMultiselectable = multiselectable
 
     if (itemIndex > -1) {
       newOpenItems.splice(itemIndex, 1)
     } else {
-      newOpenItems.push(itemId)
+      if (isMultiselectable) {
+        newOpenItems.push(itemId)
+      } else {
+        newOpenItems = []
+        newOpenItems.push(itemId)
```

Comment:
Since for this case we're always instantiating an array with a single element, if you want, I think you'd be okay to `newOpenItems = [itemId]`

Reply:
I went with a recommendation from Hana that I keep the `const` and use `.splice` to empty the array instead of straight up reassignment. Let me know if that's not preferable and I can change it back.

Reply:
I like Hana's approach more than my own! That looks great.